mlflow/telemetry/events.py

```diff
@@ -78,6 +78,70 @@ def parse(cls, arguments: dict[str, Any]) -> dict[str, Any] | None:
         return {"is_prompt": _is_prompt(tags)}
 
 
+class CreateDatasetEvent(Event):
+    name: str = "create_dataset"
+
+    @classmethod
+    def parse(cls, arguments: dict[str, Any]) -> dict[str, Any] | None:
+        try:
+            from mlflow.tracking import get_tracking_uri
+            from mlflow.utils.databricks_utils import is_databricks_default_tracking_uri
+
+            if is_databricks_default_tracking_uri(get_tracking_uri()):
+                return None
```

Comment:
I think we don't need this, if tracking uri is databricks then telemetry is disabled. You can skip this function if there're no params needing to be tracked